README.adoc

```diff
@@ -283,15 +296,15 @@ a [hotspot=docker file=0]`Docker` strategy build, which invokes the `docker buil
 from the build input. The template is parameterized with the [hotspot=appname file=0]`APP_NAME` parameter so that you can use the same
 template to create the objects for the `system` and `inventory` applications separately.
 
-Run the following commands to create the objects for the `system` and `inventory` applications:
+Run the following commands to create the objects for the `system` and `inventory` microservices:
 
 [role='command']
 ```
 oc process -f build.yaml -p APP_NAME=system | oc create -f -
 oc process -f build.yaml -p APP_NAME=inventory | oc create -f -
 ```
 
-Next, run the following commands to view the newly created image streams and build configurations for each application:
+Next, run the following commands to view the newly created `ImageStream` objects and build configurations for each microservice:
```

Comment:
suggested edit:

Next, run the following commands to view the newly created `ImageStream` objects and the build configurations for each microservice: